utils/release/bump-package.mjs

```diff
@@ -82,7 +83,10 @@ await (async () => {
   );
   const versionPrefix = `${packageJson.name}@`;
   const convention = await changelogConvention();
-  const lastTag = await getLastTag(versionPrefix);
+  const lastTag = await getLastTag({
```

Comment:
while testing the locally, I noticed that if I do:

```
git describe --tags --abbrev=0 --match="@coveo/atomic@*" release/v3
```

I'll get a

`fatal: Not a valid object name release/v3`

error if the `release/v3` branch has never been checked out locally.

Could this potentially cause issues when the action runs in the CI?

Reply:
No, because we use `fetch-depth: 0`